src/app/presentation/components/login/login.component.ts

```diff
@@ -50,10 +50,8 @@ export class LoginComponent implements OnInit {
             title: 'Login',
             text: "Success",
             showConfirmButton: false,
-            timer: 1000
+            timer: 1500
```

Comment:
Consistency 
If we displayed all messages 1000ms we shouldn't display it 1500 in one place